proto/xudrpc.proto

```diff
@@ -410,6 +425,19 @@ message Order {
   double hold = 10 [json_name = "hold"];
 }
 
+message Trade {
+  // The order id of the maker order involved in this trade.
+  Order maker_order = 1 [json_name = "maker_order"];
+  // The order id of the taker order involved in this trade, if applicable.
+  string taker_order_id = 2 [json_name = "taker_order_id"];
```

Comment:
This should also be an `Order` message like the maker order above, not just the id.